src/i18n.js

```diff
@@ -63,6 +71,10 @@ const resources = {
         '您的 WiFi 信息永远不会发送到服务端。本网站不使用追踪、分析或指纹识别。查看',
       'desc.source': '源码',
       'wifi.login': '连接 WiFi',
+      'wifi.size': '尺寸',
+      'wifi.size.small': '小的',
+      'wifi.size.medium': '中等的',
+      'wifi.size.large': '大',
```

Comment:
@Baoyuantop is it grammatically correct?

Reply:
```suggestion
      'wifi.size': '尺寸',
      'wifi.size.small': '小',
      'wifi.size.medium': '中等',
      'wifi.size.large': '大',
```

Reply:
Thanks!